TrkFilters/src/HelixFilter_module.cc

```diff
@@ -191,6 +201,61 @@ namespace mu2e
     }
     return true;
   }
+
+
+  float   HelixFilter::helixHitRatio(const HelixSeed* helix){
+
+    int      nCH = helix->hits().size();
+    int      nFaces(36); //was 36, now doing stations instead of faces
+    float    expected_hits(0);
+    float    z_step(178.); //are they really equally spaced? was 89 before
```

Comment:
Is this an algorithm parameter?  If so it should be in fcl.

Reply:
so, the model is considering the tracker as a perfect cylinder with equally spaced planes. This is actually a typo!, because z_sep = 3200. mm / nPlanes = 88.89 mm
Is it reasonable to be hardcoded?